### PR TITLE
Update and rename dependent/asian-pacific-journal-of-tropical-disease…

### PR DESCRIPTION
**Pull request by damnation333**
….csl to asian-pacific-journal-of-tropical-disease.csl

via https://forums.zotero.org/discussion/64839/asian-pacific-journal-of-tropical-disease-style-is-not-correct
Style guide: https://www.elsevier.com/journals/asian-pacific-journal-of-tropical-medicine/1995-7645/guide-for-authors
Open access examples: http://www.sciencedirect.com/science/journal/19957645/9/1

### CODE REVIEW

File: asian-pacific-journal-of-tropical-disease.csl [hunk 1]
```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Asian Pacific Journal of Tropical Disease</title>
+    <title-short>APJTD</title-short>
+    <id>http://www.zotero.org/styles/asian-pacific-journal-of-tropical-disease</id>
+    <link href="http://www.zotero.org/styles/asian-pacific-journal-of-tropical-disease" rel="self"/>
+    <link href="http://www.zotero.org/styles/european-journal-of-human-genetics" rel="template"/>
+    <link href="http://www.elsevier.com/journals/asian-pacific-journal-of-tropical-medicine/1995-7645/guide-for-authors" rel="documentation"/>
```
Comment (rmzelle) [inline asian-pacific-journal-of-tropical-disease.csl]: This "documentation" link is to a different journal ("Medicine", not "Disease").

https://www.journals.elsevier.com/asian-pacific-journal-of-tropical-disease/
https://www.journals.elsevier.com/asian-pacific-journal-of-tropical-medicine/

Are you sure they share the same format?

Comment (damnation333) [inline asian-pacific-journal-of-tropical-disease.csl]: ah man.... thanks for spotting that.
There is no guide for the APJTD actually then. 
As an elsevier journal the APJTM should then have one of their house styles, no?

Comment (rmzelle) [inline asian-pacific-journal-of-tropical-disease.csl]: > As an elsevier journal the APJTM should then have one of their house styles, no?

Well, generally speaking yes, although there are exceptions.

Comment (rmzelle) [inline asian-pacific-journal-of-tropical-disease.csl]: > There is no guide for the APJTD actually then.

Per https://forums.zotero.org/discussion/comment/272296/#Comment_272296, there seems to be a guide for the journal at http://www.apjtcm.com/authors.html.

**Comment by rmzelle** [comment]
Thanks!

**Comment by rmzelle** [comment]
(@damnation333, you'll make a single PR for the rest of the journals?)

**Comment by damnation333** [comment]
Yes. Will do the rest now, since this one got signed off. I shall find out how to do one big PR. :)

**Pull request merged**